prestodb/druid0.13.0/Dockerfile

```
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM prestodb/centos6-oj8:latest
```

this should be 

```suggestion
FROM prestodb/centos6-oj8:unlabelled
```

(this is probably why Travis fails)

---

switched to `library/centos:6.9`

---

The Java download & installation takes some time and e.g. I can be the one that will be doing the release.
I would prefer that you use `FROM prestodb/centos6-oj8:unlabelled` as the base.
If there is a problem with such approach, please let me know. You can ping me on Slack.